go/vt/vtgate/evalengine/translate.go

```diff
@@ -570,6 +570,21 @@ func translateCaseExpr(node *sqlparser.CaseExpr, lookup TranslationLookup) (Expr
 	return &result, nil
 }
 
+func translateBetweenExpr(node *sqlparser.BetweenExpr, lookup TranslationLookup) (Expr, error) {
+	// x BETWEEN a AND b => x >= a AND c <= b
```

Comment:
`c` => `x` I think here?

Reply:
see @rsajwani, you just can't trust comments... ;)